R/gtfs2gps.R

```diff
@@ -3,17 +3,22 @@
 #' @description Convert GTFS data to GPS format by sampling points using a
 #' spatial resolution. This function creates additional points in order to
 #' guarantee that two points in a same trip will have at most a given
-#' distance, indicated as a spatial resolution. This function uses progressr
-#' internally to show progress bars.
+#' distance, indicated as a spatial resolution. It is possible to use future package
+#' to parallelize the execution (or use argument plan). This function also
+#' uses progressr internally to show progress bars.
 #' 
 #' @param gtfs_data A path to a GTFS file to be converted to GPS, or a GTFS data
 #' represented as a list of data.tables.
 #' @param spatial_resolution The spatial resolution in meters. Default is 50m.
-#' @param parallel Decides whether the function should run in parallel. Defaults to FALSE.
```

Comment:
Hi Pedro. Can we can keep the parameter `parallel` ? It is much more intuitive than `plan`

Reply:
Do you mean parallel instead of plan? Sure.

Reply:
yes, thanks!